src/components/navigation/NavigationMenu.vue

```diff
@@ -82,12 +83,16 @@ export default class NavigationMenu extends Vue {
     private contextProfile: Profile | null = null;
     private LaunchMode = LaunchMode;
 
-    get thunderstoreModList() {
-        return this.$store.state.thunderstoreModList || [];
+    get thunderstoreModCount() {
```

Comment:
I'm assuming this doesn't need to
```ts
@Watch("$store.state.modFilters.showDeprecatedPackages")
```
?

Reply:
Based on manual testing it's not needed, although I must admit I'm not entirely sure why watching is not needed here while it's needed in some other places.

Reply:
Perhaps because it's a getter?

Reply:
Tried to turn it into a regular method and stuff still worked 😔

Reply:
Weird, get makes it a computed method and so it makes sense that you don't need to watch. As a regular method I'm not sure, I'd have expected a watch or something 🤔